### PR TITLE
v1.0.4

### PR DESCRIPTION
**Pull request by jcvernaleo**

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -1,6 +1,53 @@
+# [v1.0.4](https://github.com/decred/decred-binaries/releases/tag/v1.0.4)
+
+## 2017-06-09
+
+This release contains a fix for an error that caused Paymetheus to be unable to talk to the
+dcrwallet process.  Occasionally, HTTP/2 parsing error messages would be displayed and Paymetheus
+would need to be closed.  Under certain instances it was possible that the wallet would begin to work
+after restarting, but some users continued to hit the issue even after restarting.  The bug was caused
+by an integer overflow in the grpc-go version imported by dcrwallet 1.0.3 and was fixed by upgrading
+the grpc-go dependency to the latest 1.4.0 release.
```
Comment (jolan) [inline release-notes.md]: 1.0.4

Comment (jrick) [inline release-notes.md]: no, this is right.  that is the version of the grpc-go dep

Comment (jolan) [inline release-notes.md]: oops nm, that is the grpc version

Comment (jrick) [inline release-notes.md]: https://github.com/grpc/grpc-go/releases/tag/v1.4.0

**Pull request merged**